### PR TITLE
*: add and report store labels to PD

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

File: src/util/config.rs [hunk 1]
```diff
@@ -130,6 +136,35 @@ pub fn parse_readable_int(size: &str) -> Result<i64, ConfigError> {
     }
 }
 
+pub fn parse_store_labels(labels: &str) -> Result<HashMap<String, String>, ConfigError> {
+    let mut map = HashMap::new();
+
+    let re = Regex::new(r"^[a-z0-9][a-z0-9-._]*[a-z0-9]$").unwrap();
```
Comment (siddontang) [inline src/util/config.rs]: do we need to create the regex every time?

**Comment by siddontang** [comment]
Rest LGTM

**Comment by huachaohuang** [comment]
PTAL @disksing

File: src/util/config.rs (hunk 1 above)
Comment (disksing) [inline src/util/config.rs]: Single character is not allowed?

**Comment by disksing** [comment]
LGTM

**Pull request merged**